react/components/SearchBar/components/ResultsList.js

```diff
@@ -1,116 +1,122 @@
       query = `map=c,ft&rest=${terms.slice(1).join(',')}`
     }
+
     return { page, params, query }
   }
 
-  renderSpinner() {
+  const renderSpinner = () => (
+    <div className="w-100 flex justify-center">
+      <div className="w3 ma0">
+        <Spinner />
+      </div>
+    </div>
+  )
+
+  if (data.loading) {
     return (
-      <div className="w-100 flex justify-center">
-        <div className="w3 ma0">
-          <Spinner />
-        </div>
+      <div className={listClassNames}>
+        <div className={listItemClassNames}>{renderSpinner()}</div>
       </div>
     )
   }
 
-  render() {
-    const {
-      data,
-      emptyPlaceholder,
-      inputValue,
-      closeMenu,
-      onClearInput,
-    } = this.props
-    const items = data.autocomplete ? data.autocomplete.itemsReturned : []
-    if (data.loading) {
-      return (
-        <div className={listClassNames}>
-          <div className={listItemClassNames}>{this.renderSpinner()}</div>
-        </div>
-      )
-    }
-
-    if (!items.length) {
-      return (
-        <div className={listClassNames}>
-          <div className={listItemClassNames}>{emptyPlaceholder}</div>
-        </div>
-      )
-    }
-
+  if (!items.length) {
     return (
       <div className={listClassNames}>
-        <Link
-          onClick={() => {
-            onClearInput()
-            closeMenu()
-          }}
-          page="store.search"
-          params={{ term: inputValue }}
-          query="map=ft"
-          className={listItemClassNames}
-        >
-          {inputValue}
-        </Link>
-
-        {items.map((item, index) => {
-          return (
-            <Fragment key={item.name + index}>
-              <hr className="o-05 ma0 w-90 center" />
-              <Link
-                onClick={() => {
-                  onClearInput()
-                  closeMenu()
-                }}
-                {...this.getLinkProps(item)}
-                className={listItemClassNames}
-              >
-                {item.thumb && (
-                  <img
-                    className={`${styles.resultsItemImage} mr4`}
-                    src={getImageUrl(item.thumb)}
-                  />
-                )}
-                <div className="flex justify-start items-center">
-                  {item.name}
-                </div>
-              </Link>
-            </Fragment>
-          )
-        })}
+        <div className={listItemClassNames}>{emptyPlaceholder}</div>
       </div>
     )
   }
+
+  return (
+    <div className={listClassNames}>
+      <Link
+        onClick={() => {
+          onClearInput()
+          closeMenu()
+        }}
+        page="store.search"
+        params={{ term: inputValue }}
+        query="map=ft"
+        className={listItemClassNames}
+      >
+        {inputValue}
+      </Link>
+
+      {items.map((item, index) => {
+        return (
+          <Fragment key={item.name + index}>
+            <hr className="o-05 ma0 w-90 center" />
```

Comment:
perhaps use a color token here, instead of relying on the default color and transparency